optimum/exporters/tasks.py

```diff
@@ -884,6 +885,11 @@ class TasksManager:
             "masked-im",
             onnx="SwinOnnxConfig",
         ),
+        "swin2sr": supported_tasks_mapping(
+            "feature-extraction",
+            "image-to-image",
+            onnx="SwinOnnxConfig",
```

Comment:
Could you just add a

```
class Swin2srOnnxConfig(SwinOnnxConfig):
    pass
```

in `model_configs.py`? It's not particularly useful but we so far kept different classes for each arch in case methods need to be overridden in the future.

Reply:
Done! From my perspective it is fine if you just push these small additions directly to the branch, you should be to do that right?